docs/variants/asus_kgpe_d16/dasharo-compatibility/flashrom_wp_test_specification.md

```diff
@@ -0,0 +1,220 @@
+# Flashrom Write Protection – Test Specification
+
+## Test cases
+
+### WPR001.001 BIOS WP enable test
+
+#### Test description
+
+This test aims to verify if it's possible to set the write protection and
+whether it actually works.
+
+#### Test configuration data
+
+1. `FIRMWARE` = coreboot
+
+2. `OPERATING_SYSTEM` = Ubuntu 20.04
+
+#### Test setup
+
+1. Proceed with the [Generic test setup: firmware](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#firmware)
+
+2. Proceed with the [Generic test setup: OS installer](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-installer)
+
+3. Proceed with the [Generic test setup: OS installation](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-installation)
+
+4. Proceed with the [Generic test setup: OS boot from disk](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-boot-from-disk)
+
+5. Install flashrom tools and cbfstool with the below commands:
+
+    - In order to build flashrom we will need some packages and libraries.
+For Debian based distros execute:
+
+            `sudo apt-get install git make binutils build-essential ca-certificates \`
+            `libpci-dev libftdi-dev libusb-1.0-0-dev`
+
+    - Now clone the flashrom repository and fetch the patchset:
+
+            `git clone https://github.com/flashrom/flashrom`
+            `cd flashrom`
+            `git fetch https://review.coreboot.org/flashrom refs/changes/13/59713/7 && \`
+            `git checkout FETCH_HEAD`
+
+    - Build flashrom:
+
+            `make`
+
+6. Ensure the WP pin jumper on the flash is not placed.
+
+#### Test steps
+
+1. Clear SPI write protection:
+
+        `./flashrom -p internal --wp-disable`
+        `./flashrom -p internal --wp-range=0,0`
+
+2. Check protection status:
+
+        `./flashrom -p internal --wp-status`
+
+3. See what protection ranges are available for the chip:
+
+        `./flashrom -p internal --wp-list`
+
+4. Set protection range e.g.:
```

Comment:
At this point we should add reference to document in which we describe how to choose the correct write protection range